### PR TITLE
Make new PostgreSQL backend tests pass

### PR DESCRIPTION
**Pull request by rumyantseva**
# Description

After or insted of #3511.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [x] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: integration/setup/client.go [hunk 1]
```diff
@@ -39,9 +39,11 @@ func makeClient(ctx context.Context, uri string) (*mongo.Client, error) {
 		return nil, lazyerrors.Error(err)
 	}
 
-	if err = client.Ping(ctx, nil); err != nil {
-		return nil, lazyerrors.Error(err)
-	}
+	// The new PostgreSQL backend's tests can't survive this ping, as for every ping ANALYZE is called.
```
Comment (rumyantseva) [inline integration/setup/client.go:42]: That's an important moment in this PR. Unfortunately, with `ANALYZE` being called, tests stuck.

Comment (AlekSi) [reply integration/setup/client.go:42]: That's not correct anymore

File: integration/update_field_test.go [hunk 2]
```diff
@@ -363,6 +363,8 @@ func TestUpdateCommandUpsert(t *testing.T) {
 }
 
 func TestUpdateFieldErrors(t *testing.T) {
+	setup.SkipForNewPg(t, "TODO")
```
Comment (AlekSi) [inline integration/update_field_test.go:366]: TODO?

Comment (rumyantseva) [reply integration/update_field_test.go:366]: oh, I missed this one 😅

**Review by noisersup — APPROVED** [review]
Apart minor comments above LGTM

**Review by AlekSi — APPROVED** [review]

**Pull request merged**